### PR TITLE
Add zsh-bench

### PR DESCRIPTION
**Pull request by romkatv**
# Description

Add [zsh-bench](https://github.com/romkatv/zsh-bench).

# Type of changes

- [ ] A link to an external resource like a blog post
- [ ] Add/remove/update a link to a framework
- [ ] Add/remove/update a link to a plugin
- [ ] Add/remove/update a link to a tab completion
- [ ] Add/remove/update a link to a theme
- [ ] Text cleanups/typo fixes
- [x] A link to [zsh-bench](https://github.com/romkatv/zsh-bench)

# Copyright Assignment

- [x ] This document is covered by the [BSD License](https://github.com/unixorn/awesome-zsh-plugins/blob/master/LICENSE), and I agree to contribute this PR under the terms of the license.

# Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/unixorn/awesome-zsh-plugins/blob/main/Contributing.md) document.
- [x] I have confirmed that the link(s) in my PR are valid.
- [x] My entries are single lines and are in the appropriate (plugins, themes or completions) section, and in alphabetical order in their section.
- [ ] All new and existing tests passed.
- [ ] Any added completions have a readme and a license file in their repository.
- [ ] Any added frameworks have a readme and a license file in their repository.
- [ ] Any added plugins have a readme and a license file in their repository.
- [ ] Any added themes have a screen shot, a readme and a license file in their repository.
- [ ] I have stripped leading and trailing **zsh-**, **zsh-plugin** and/or **oh-my-zsh-** substrings from the link name. This makes it easier to find plugins/themes/completions by name by preventing big clusters in the **O** and **Z** sections of the list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2015,6 +2015,7 @@ I've documented some recommendations for writing a new plugin [here](https://git
 ### ZSH Tools
 
 * [ShellSpec](https://github.com/shellspec/shellspec) - A full-featured BDD unit testing framework for dash, bash, ksh, ZSH and all POSIX shells.
+* [zsh-bench](https://github.com/romkatv/zsh-bench) - A benchmark for interactive ZSH
```
Comment (unixorn) [inline README.md:2018]: Please add a `.` on the end of the description, otherwise looks good

**Review by unixorn — CHANGES_REQUESTED** [review]

**Comment by unixorn** [comment]
Please signoff on your commits - the DCO github action details include instructions.

File: README.md (hunk 1 above)
Comment (romkatv) [reply README.md]: Done.

Two out of tree existing elements in this list don't have trailing periods, so I went with the majority. Now we have a nice 50/50 split \o/

**Review by unixorn — APPROVED** [review]
Thanks for the contribution!

File: README.md (hunk 1 above)
Comment (unixorn) [reply README.md]: I'll fix the others, thanks for the heads up

**Comment by romkatv** [comment]
> Please signoff on your commits - the DCO github action details include instructions.

Done.

The number of layers of red tape is impressive!

**Pull request merged**